### PR TITLE
Gradle upgrade

### PR DESCRIPTION
**Pull request by ekoby**

### CODE REVIEW

File: ziti/src/test/kotlin/org/openziti/net/nio/AsyncTLSChannelTest.kt [hunk 1]
```diff
@@ -46,7 +43,7 @@ class AsyncTLSChannelTest {
     lateinit var ch: AsyncTLSChannel
 
     @Rule
-    @JvmField val timeout = Timeout.seconds(1500)
+    @JvmField val timeout = Timeout.seconds(15)
```
Comment (dovholuknf) [inline ziti/src/test/kotlin/org/openziti/net/nio/AsyncTLSChannelTest.kt:46]: just confirming 1500 -> 15 is intentional?

File: .github/workflows/gradle-build.yml [hunk 2]
```diff
@@ -14,11 +14,21 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Set up JDK
+        uses: actions/setup-java@v2
         with:
-          java-version: 1.8
+          java-version: 11
```
Comment (dovholuknf) [inline .github/workflows/gradle-build.yml:20]: out of curiosity - why not 14? android reasons or other?

**Review by dovholuknf — APPROVED** [review]
looks good assuming 1500 -> 15 is anticipated

File: .github/workflows/gradle-build.yml (hunk 2 above)
Comment (ekoby) [reply .github/workflows/gradle-build.yml:20]: java 11 is the only LTS until 17

**Pull request merged**